cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp

```diff
@@ -439,13 +439,13 @@ int LuaEngine::handleCommonEvent(void* data)
         return 0;
    
     CommonScriptData* commonInfo = static_cast<CommonScriptData*>(data);
-    if (NULL == commonInfo->eventName || 0 == commonInfo->handler)
+    if ('\0' == commonInfo->eventName[0] || 0 == commonInfo->handler)
```

Comment:
What's the warning of this?
XMLHttpRequest & WebSocketTest in lua test will not get response after this change.